### DISCUSSION TITLE
Problems with Science Lab Towers

### DISCUSSION THREAD

**Discussion by FlameStormer2000** [cataclysmbn/Cataclysm-BN#2625] [General]
I think Lab Towers are a really interesting location in theory. Even through they're basically just a copy paste of generic lab tiles they at least have location novelty/variety value. They have some issues that I'd like to work towards addressing.

- On a megacity world (size=16/distance=1) I checked a 5x5 overmap grid and found 12 Lab Towers. To me this feels super low, anecdotal but on default world settings (size=8/distance=4) I have never found a lab tower despite flying around looking for them. You basically trip over standard labs but tower labs seem like an odyssey alone to try and find. 

- Top floor doesn’t have a roof. It gens the standard underground ceiling but this often leaves gapping holes that could let a player just drop in from above.

- Towers occasional gen with a collapse/cave-in tile.

Putting up a thread to keep track and see if anyone else has noticed other issues or has suggestions to add. Personally I'd love to give lab towers a unique overhaul with special themed sets of tiles but that's a pipe dream at this point.

**Comment by Zlorthishen** [comment]
there is discountinuty between level 0 and level 1 science tower.
- level 0, ground floor
![cataclysm-tiles_2023-04-16_21-39-06](https://user-images.githubusercontent.com/79779913/232364893-b056deed-3738-49bd-91b8-48c854392a3d.png)

- level 1, first floor of science lab.
![cataclysm-tiles_2023-04-16_21-35-51](https://user-images.githubusercontent.com/79779913/232364928-5802dca5-d1de-4198-a41a-b0516c153595.png)
